internal/consensus/invalid_test.go

```diff
@@ -26,13 +26,13 @@ func TestReactorInvalidPrecommit(t *testing.T) {
 
 	config := configSetup(t)
 
-	n := 4
+	n := 2
```

Comment:
```suggestion
	const n = 2
```
Since we're in here anyway